submitter/tx_submitter.go

```diff
@@ -39,6 +42,18 @@ func (s *TxSubmitter) SubmitTransactionLoop() {
 	s.cachedEventHash = make(map[uint64][]byte, common.CacheSize)
 	submitLoopCount := 0
 	for {
+		for {
+			// blsPubKey of current submitter
+			blsPubKey, err := s.executor.QueryInturnAttestationSubmitter()
```

Comment:
If there is no challenge to attest, should we still do the loop?
Can we
1) move the code to where we want to submit attestation
2) add some interval between the loop, otherwise it will bring a big burden the the node.

Reply:
fixed